bin/devrun.dart

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env dart
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Runs io.js with dev_compiler's generated code.
+library dev_compiler.bin.devrun;
+
+import 'dart:io';
+
+import 'package:dev_compiler/src/options.dart';
+import 'package:dev_compiler/src/runner/file_utils.dart' show turnFilesToUris;
+import 'package:dev_compiler/src/runner/runtime_utils.dart' show
+    listRuntimeFiles, listOutputFiles, getRuntimeFileAlias, getMainModuleName;
+import 'package:dev_compiler/src/runner/iojs_utils.dart' show
+    checkIoJsVersion, startIoJs;
+
+import 'package:path/path.dart';
+
+import 'devc.dart' show validateOptions, compile;
```

Comment:
interesting. don't usually see importing another bin/ script. But I can't think of anything wrong with it, so maybe it's a valid pattern. it can be hard(er) to test "bin/" though, so we usually keep them really small, with most of the logic someplace else (e.g. lib/src/compiler.dart)